css/documentation.css

```diff
@@ -1,41 +1,81 @@
-.show {
-  display: none;
+/* Component Section In Documentation Styling */
+.component-section {
+  flex-grow: 1;
 }
 
-.NoShow {
-  display: block;
+.container {
+  margin: auto;
+  padding: 2rem;
 }
 
-.page-container {
-  display: flex;
-  width: 80%;
+.component-heading {
+  font-size: 3rem;
+  font-weight: 700;
+  margin: 0 0 1rem 0;
 }
 
-.section {
-  margin: 3rem;
-  padding: 0 1rem;
+.component-description {
+  font-size: 1rem;
+  line-height: 1.5rem;
+  font-weight: 400;
+  margin: 0 0 2rem 0;
 }
 
+.component-container {
+  display: flex;
+  justify-content: center;
+  width: 100%;
+  background-color: var(--container-background-color);
+  min-height: 9rem;
+  align-items: center;
+  flex-wrap: wrap;
+  margin: 0 0 2rem 0;
+  padding: 1rem;
+}
 
-/* For Smaller Screen  */
+.card-component-container {
+  display: flex;
+  justify-content: center;
+  width: 100%;
+  background-color: var(--container-background-color);
+  min-height: 9rem;
+  align-items: center;
+  flex-wrap: wrap;
+  margin: 0 0 2rem 0;
+  padding: 1rem;
+}
 
-@media only screen and (max-width: 768px){
+.navigation-component-container {
+  width: 100%;
+  background-color: var(--container-background-color);
+  min-height: 9rem;
+  margin: 0 0 2rem 0;
+  padding: 1rem;
+}
 
-    .show{
-        display: block;
-    }
+.alert-box {
+  flex-direction: column;
+}
 
-   .NoShow{
-        display: none;
-    }
+.component-subheading {
+  font-size: 1.5rem;
+  font-weight: 700;
+  margin: 0 0 1rem 0;
+}
 
-    .logo{
-        margin: 0 5rem;
-    }
+.component-container img {
+  margin: 0 0.625rem;
+}
 
-    .page-container{
-        display: block;
-        width: 80%;
-    }
+.component-code {
+  display: flex;
```

Comment:
add helper class for flex

Reply:
Will do that afterwards as it required lot of refactoring in code